src/server/pps/cmds/cmds.go

```diff
@@ -51,6 +53,30 @@ const (
 	secrets   = "secrets"
 )
 
+// copied from Client, because need to extend for projects without breaking user facing API.
```

Comment:
I would change the signature to `(context.Context,  pps.APIClient, func(*pps.JobInfo) error) error`, letting the caller pull stuff out of the `*client.APIClient`.

Reply:
Changed to `func listJobFilterF(ctx context.Context, client pps.API_ListJobClient, f func(*pps.JobInfo) error) error`